htslib/bgzf.h

```diff
@@ -416,6 +417,16 @@ typedef struct BGZF BGZF;
     int bgzf_index_dump_hfile(BGZF *fp, struct hFILE *idx,
                               const char *name) HTS_RESULT_USED;
 
+
+    /*
+     * Buffers up arguments to hts_idx_push for later use, once we've written all bar
+     * this block.  This is necessary when multiple blocks are in flight (threading).
+     *
+     * Returns 0 on success,
+     *        -1 on failure
+     */
+    int bgzf_idx_push(BGZF *fp, hts_idx_t *hidx, int tid, int beg, int end, uint64_t offset, int is_mapped);
```

Comment:
Is this function for public use? If not, it could go in _hts_internal.h_ or so.

Reply:
Good point.  Thanks.